portal/db/models/models.go

```diff
@@ -139,9 +139,9 @@ type (
 
 		UserID     ID     `gorm:"not null;unique"`
 		User       *User  `gorm:"foreignkey:UserID"`
-		UnionId    string `gorm:"type:varchar(128);not null;unique"`
+		UnionID    string `gorm:"type:varchar(128);not null;unique"`
 		Nickname   string `gorm:"type:varchar(64);not null;unique"`
-		HeadImgUrl string `gorm:"type:varchar(512)"`
+		HeadImgURL string `gorm:"type:varchar(512)"`
```

Comment:
I am not sure whether will this change affect database schema.

Reply:
Should be fine. AFAIK MySQL column names are not case sensitive.